.env

```diff
@@ -1 +1,42 @@
-ELK_VERSION=7.15.0
+ELASTIC_VERSION=8.15.0
```

Comment:
Since, all our mappings and current configs are base on v7.15.0. So, Are we sure we want to use the latest version on local env?

IMO, it's always better to keep the same version on all of the envs

Reply:
okay Rajendra